client/src/components/Modal/MapModal.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Modal, TouchableOpacity, Dimensions, TouchableWithoutFeedback, Text } from 'react-native';
+
+import MapView, { Marker } from 'react-native-maps';
+import { IconMarker } from '../../components/MapMarker/';
+
+import styles from './styles';
+
+const screen = Dimensions.get('window');
+
+const ASPECT_RATIO = screen.width / screen.height;
+const LATITUDE_DELTA = 0.0922;
+const LONGITUDE_DELTA = LATITUDE_DELTA * ASPECT_RATIO;
+
+const MapModal = ({
+  visible,
+  title,
+  initialRegion,
+  closeModal,
+  backModal,
+  markers,
+}) => (
+  <View>
+    <Modal
+      transparent
+      visible={visible}
+      onRequestClose={closeModal}
+      animationType="fade"
+    >
+      <TouchableWithoutFeedback onPress={() => closeModal}>
```

Comment:
@sdunster how did you let that slip past!

we should have a linter rule to catch these

Reply:
ooh yeah ouch. hard to lint on this stuff because that might be entirely sensible way of doing things